### PR TITLE
UIU-2262 create Jest/RTL test for CreateResetPasswordControl.js

### PR DESCRIPTION
**Pull request by Mathansivavelmurugesan**
Code coverage and unit testing for CreateResetPasswordControl Component.

### CODE REVIEW

**Review by mkuklis — APPROVED** [review]

File: src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    expect(screen.getByText('testemail@email.com')).toBeInTheDocument();
+  });
+  it('If it redirects after POST fails', async () => {
+    const mockFunc = jest.fn(() => new Promise((_, reject) => {
+      const error = 'Something Went Wrong';
+      reject(error);
+    }));
+    await waitFor(() => renderCreateResetPasswordControl(propData(mockFunc)));
+    await userEvent.click(screen.getByText('ui-users.extended.sendResetPassword'));
+    await screen.debug();
```
Comment (zburke) [inline src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js:52]: It's odd to me that there is no assertion in this last block. Is there nothing visible in the UI to indicate the POST request fails? If not, then I think we should comment this test for now and file a new UIU ticket to surface the error message.

File: src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    renderCreateResetPasswordControl(propData(mockFunc));
+    expect(screen.getByText('ui-users.extended.sendResetPassword')).toBeInTheDocument();
+  });
+  it('If reset password renders on click', async () => {
+    const mockFunc = jest.fn(() => new Promise((resolve, _) => {
+      resolve({ ok: true, link: 'bl-users/password-reset/link' });
+    }));
+    await waitFor(() => renderCreateResetPasswordControl(propData(mockFunc)));
+    await userEvent.click(screen.getByText('ui-users.extended.sendResetPassword'));
+    expect(screen.getByText('testemail@email.com')).toBeInTheDocument();
```
Comment (zburke) [inline src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js:43]: Are you confident this assertion is only valid after the click? Given the user-edit form is populated, I would expect the email address to be in the document both before and after clicking the button. Is there a different/better value we can assert on, i.e. one that is dependent on the click?

**Review by zburke — COMMENTED** [review]
It isn't clear to me that these tests are specific enough.

File: src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js (hunk 2 above)
Comment (Mathansivavelmurugesan) [reply src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js]: Changed the assertion to copylink which would be rendered only on the click

**Pull request merged**